addons/website/views/snippets/s_blockquote.xml

```diff
@@ -2,14 +2,18 @@
 <odoo>
 
 <template name="Blockquote" id="s_blockquote">
-    <blockquote class="s_blockquote s_blockquote_classic w-100 mx-auto o_animable blockquote">
-        <i class="s_blockquote_icon fa fa-1x fa-quote-left bg-o-color-2 rounded"/>
-        <div class="s_blockquote_content bg-100">
-            <p>Write a quote here from one of your customers. Quotes are a great way to build confidence in your products or services.</p>
-            <footer class="blockquote-footer">
-                <img src="/web/image/website.s_blockquote_default_image" class="s_blockquote_avatar img rounded-circle me-2" alt=""/>
-                <span class="s_blockquote_author text-muted"><b>John DOE</b> &#8226; CEO of MyCompany</span>
-            </footer>
+    <blockquote class="s_blockquote s_blockquote_minimalist s_blockquote_small o_cc o_cc2 o_animable position-relative d-flex flex-column gap-4 w-100 mx-auto p-4 fst-normal" data-vcss="001">
+        <div class="s_blockquote_minimalist_border position-absolute top-50 start-0 translate-middle-y h-100 bg-o-color-1"/>
+        <i class="s_blockquote_icon fa fa-quote-right bg-o-color-1 position-absolute top-0 start-50 translate-middle rounded" role="img"/>
```

Comment:
The icon alignment controller has no effect here 😉
It would be interesting to move the icon, wdyt?
![image](https://github.com/user-attachments/assets/ffc5228a-cbd3-487d-9e46-e3681ef209a1)
![image](https://github.com/user-attachments/assets/8a252790-5119-467f-baac-b73d54195517)